schwifty/checksum/netherlands.py

```diff
@@ -0,0 +1,24 @@
+import functools
+
+from schwifty import checksum
+from schwifty import registry
+
+
+register = functools.partial(checksum.register, prefix="NL")
+
+
+@register
+class DefaultAlgorithm(checksum.Algorithm):
+    name = "default"
+    accepts = checksum.InputType.BBAN
```

Comment:
If you're actually just interested in the account code, you can also use `InputType.ACCOUNT_CODE`.

Reply:
for this and others, I'd be glad if you could edit it, I'm not very familiar with this codebase, just had the "rare" NL algo and wanted to help out. I copied BE checksum file and edited that.